tools/binlog/deploy.md

```diff
@@ -0,0 +1,494 @@
+
+        db-type 设置为 "mysql"， 配置下游 MySQL 信息。
+
+        ```toml
+        # downstream storage, equal to --dest-db-type
+        # Valid values are "mysql", "file", "kafka", "flash".
+        db-type = "mysql"
+
+        # the downstream MySQL protocol database
+        [syncer.to]
+        host = "172.16.10.72"
+        user = "root"
+        password = "123456"
+        port = 3306
+        # Time and size limits for flash batch write
+        # time-limit = "30s"
+        # size-limit = "100000"
+        ```
+
+    - 以下游为增量备份文件为例
+
+        ```bash
+        $ cd /home/tidb/tidb-ansible/conf
+        $ cp drainer.toml drainer_pb_drainer.toml
+        $ vi drainer_pb_drainer.toml
+        ```
+
+        db-type 设置为 "file"。
+
+        ```toml
+        # downstream storage, equal to --dest-db-type
+        # Valid values are "mysql", "file", "kafka", "flash".
+        db-type = "file"
+
+        # Uncomment this if you want to use `file` or `sql` as `db-type`.
+        # `Compress` compresses the output file, like the `file` and `sql` file. Now it supports the `gzip` algorithm only. 
+        # The value can be `gzip`. Leave it empty to disable compression. 
+        [syncer.to]
+        compression = ""
+        # default data directory: "{{ deploy_dir }}/data.drainer"
+        dir = "data.drainer"
+        ```
+
+4. 部署 Drainer
+
+    ```bash
+    $ ansible-playbook deploy_drainer.yml
+    ```
+
+5. 启动 Drainer
+
+    ```bash
+    $ ansible-playbook start_drainer.yml
+    ```
+
+## 使用 Binary 部署 TiDB-Binlog
+
+### 下载官方 Binary
+
+```bash
+wget https://download.pingcap.org/tidb-{version}-linux-amd64.tar.gz
+wget https://download.pingcap.org/tidb-{version}-linux-amd64.sha256
+
+# 检查文件完整性，返回 ok 则正确
+sha256sum -c tidb-{version}-linux-amd64.sha256
+```
+
+对于 v2.1.0 GA 及以上版本，Pump 和 Drainer 已经包含在 TiDB 的下载包中，其他版本需要单独下载 Pump 和 Drainer:
+
+```bash
+wget https://download.pingcap.org/tidb-binlog-{version}-linux-amd64.tar.gz
+wget https://download.pingcap.org/tidb-binlog-{version}-linux-amd64.sha256
+
+# 检查文件完整性，返回 ok 则正确
+sha256sum -c tidb-binlog-{version}-linux-amd64.sha256
+```
+### 使用样例
+假设有三个 PD，一个 TiDB，另外有两台机器用于部署 Pump，一台机器用于部署 Drainer。各个节点信息如下：
+
+```
+TiDB="192.168.0.10"
+PD1="192.168.0.16"
+PD2="192.168.0.15"
+PD3="192.168.0.14"
+Pump="192.168.0.11"
+Pump="192.168.0.12"
+Drainer="192.168.0.13"
+```
+
+下面以此为例，说明 Pump/Drainer 的使用。
+
+1. 使用 binary 部署 Pump
+
+    - Pump 命令行参数说明（以在 “192.168.0.11” 上部署为例）
+
+        ```bash
+        Usage of Pump:
+        -L string
+            日志输出信息等级设置：debug，info，warn，error，fatal (默认 "info")
+        -V
+            打印版本信息
+        -addr string
+            Pump 提供服务的 RPC 地址(-addr="192.168.0.11:8250")
+        -advertise-addr string
+            Pump 对外提供服务的 RPC 地址(-advertise-addr="192.168.0.11:8250")
+        -config string
+            配置文件路径，如果你指定了配置文件，Pump 会首先读取配置文件的配置；
+            如果对应的配置在命令行参数里面也存在，Pump 就会使用命令行参数的配置来覆盖配置文件里的配置。
+        -data-dir string
+            Pump 数据存储位置路径
+        -enable-tolerant
```

Comment:
already abandon this config

Reply:
deleted in https://github.com/pingcap/docs-cn/pull/1264/commits/98c62994c644c19dd9793ea34680b4e29c1cebe3